.gitignore

```diff
@@ -8,3 +8,4 @@ logs
 .DS_Store
 test-results.xml
 .env
+.idea
```

Comment:
that's confusing... are you using an old version? later versions of Webstorm place a `.gitignore` file

![Screenshot 2024-01-18 at 13 42 48](https://github.com/adobe/spacecat-audit-scheduler/assets/1872195/2962c85d-7209-4243-80b3-9a353b12680f)

Reply:
No, my IDE is up-to-date and I've just stumbled upon this today, hence the `.gitignore` update.